README.md

```diff
@@ -2,6 +2,13 @@
 
 **This is not an officially supported Google product.**
 
-This project is aimed to create a visualizer application for segmented timeseries data.
+This project is aimed to create a visualizer application for segmented
+timeseries data.
+
+Time series forecasting is a pretty important area of machine learning. While we
+invest lots of time to come up with a good model we often neglect the necessity
+of a good visualization tool that helps us to to investigate a forecast in more
+details. The aim of this project is to offer a standalone application and a
+common infrastructure that would help us to visualize and investigate general
```

Comment:
infrastructure is in general uncountable, so indefinite article should be omitted

Reply:
I would also say '_the_ general time series problem', but I'm not sure here